arm-web/2016-08-01/swagger/WebApps.json

```diff
@@ -8931,8 +9066,57 @@
           }
         ],
         "responses": {
-          "204": {
-            "description": "No Content"
+          "200": {
+            "description": "OK"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/resourceHealthMetadata": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Gets the category of ResourceHealthMetadata to use for the given site",
+        "description": "Gets the category of ResourceHealthMetadata to use for the given site",
+        "operationId": "WebApps_GetResourceHealthMetadataSlot",
```

Comment:
Please provide x-ms-examples section according to https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md.